src/materials.rs

```diff
@@ -19,18 +22,24 @@ impl Material {
             diffuse: 0.9,
             specular: 0.9,
             shininess: 200.,
+            pattern: None,
         }
     }
 
     pub fn lighting(
         material: Material,
```

Comment:
Maybe we should remove this and get the material from the shape.

Reply:
Agreed, but the book does it this way, at least in the current chapter. Not sure if it will refactor this function more later.

Reply:
Ah, ok. Lets keep it as it is then.